crates/wasmtime/src/compile/code_builder.rs

```diff
@@ -119,6 +127,8 @@ impl<'a> CodeBuilder<'a> {
             .with_context(|| format!("failed to read input file: {}", file.display()))?;
         self.wasm = Some(wasm.into());
         self.wasm_path = Some(file.into());
+        self.dwarf_package_from_wasm_path()?;
```

Comment:
Could you update the documentation to indicate that the `*.dwp` file is probed for?